packages/core/src/zip/selector.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
+
+import R from 'ramda'
+
+import * as zip from './impl'
+
+import { isOfKind } from '../data'
+
+import { isStringArray } from './select'
+
+// Selector fns
+// - A selector fn has the signature: (...args, loc) => [loc]
+// - It takes a number of arguments that determine its behavior
+// - a zipper location at the end
+// - and will return a an array of locations that have been selected by its execution
+
+const isNotChildCollection = R.complement(isOfKind('@@skele/child-collection'))
+
+// children :: Location -> [Location]
+export const elementChildren = loc => elementChildrenFor(null, loc)
+
+// Key = String | [String]
+// childrenFor :: Key -> Location -> [Location]
+export const elementChildrenFor = R.curry((key, loc) => {
```

Comment:
This should be just called `children` or maybe `childrenUnder` or `childrenAt`. It will be the most frequently used selector:

```javascript
render() {
  const { uiFor } = this.props
  return uiFor(elementChildrenFor('bla')) // this is considerably more ugly than just uiFor('bla')
}

Also, this is an element-zipper specific function, not a general one... We surely will have those, but a different namespace may make more sense (e.g. ancestors and descendants are general for all zippers, this one isn't).

Reply:
I've created a new namespace. Can you check it out and let me know if it is OK?
https://github.com/netceteragroup/skele/pull/113/commits/ae0798e69c995c1e1e780d92eb4c86a9d95e7c2f

I've renamed the functions to `children` and `childrenAt`